website/docs/developer/api/api.md

```diff
@@ -417,7 +511,7 @@ The following table lists the fields in the output:
 The API returns status code 200 on success, 400 if `jobId` or `pageIndex` are missing and 500 if an error occurs.
 
 ## Internal Reconcile report mismatch API
-The `orca/datamanagement/reconciliation/internal/jobs/job/{jobid}/mismatches` API call receives job id and page index from end user and returns reporting information of files that are missing from ORCA S3 bucket or have different metadata values than what is expected. Note that `{jobid}` is optional.
+The `orca/datamanagement/reconciliation/internal/jobs/job/{jobid}/mismatches` API call receives job id and page index from end user and returns reporting information of files that are missing from ORCA S3 bucket or have different metadata values than what is expected.
```

Comment:
Is jobid no longer optional?

Reply:
It was optional for requesting the status of a job, but I don't believe it was ever optional for retrieving reports. Likely a copy/paste issue when replicating the job-status docs.